tests/e2e/utils.js

```diff
@@ -35,17 +43,12 @@ export async function getExtensionId() {
   return extensionId;
 }
 
-const SCHEME =
-  browser.capabilities.browserName === 'chrome'
-    ? 'chrome-extension'
-    : 'moz-extension';
-
 export async function getExtensionPageURL(page, file = 'index.html') {
-  return `${SCHEME}://${await getExtensionId()}/pages/${page}/${file}`;
+  return `${browser.isChromium ? 'chrome-extension' : 'moz-extension'}://${await getExtensionId()}/pages/${page}/${file}`;
 }
 
 export function getExtensionElement(id) {
-  return $(`[data-qa="${id}"]`);
+  return $(`>>>[data-qa="${id}"]`);
```

Comment:
what does `>>>` do?

Reply:
Ensure that the selector goes into the Shadow DOM